Sources/CardPayments/CardVaultDelegate.swift

```diff
@@ -17,4 +17,19 @@ public protocol CardVaultDelegate: AnyObject {
     ///   - client: the CardClient associated with delegate
     ///   - didFinishWithError: the error returned by the Card vault flow
     func card(_ cardClient: CardClient, didFinishWithVaultError vaultError: CoreSDKError)
+
+    /// Notify that the Card vault flow has been cancelled
+    /// - Parameters:
+    ///   - client: the CardClient associated with delegate
+    func cardDidCancelVault(_ cardClient: CardClient)
```

Comment:
This method will only get called if a 3DS flow was cancelled. To match the naming convention below, it should include `threeDSecure` in the name IMO

Reply:
Also it doesn't need `Vault` in it, since that's already in the class name

Reply:
I guess it should be `cardThreeDSecureDidCancel`

Reply:
Not `cardDidCancelThreeDSecure`?

Reply:
🤣 I just changed it from that. because other 3ds functions are `cardThreeDSecureDidFinish`, etc

Reply:
So I think Sammy's point was that this cancel path doesn't exist for non 3DS contingency flow.

Reply:
Oh got it yeah if there's already a convention in place it's best to match that.

Objectively though it has somewhat of a yoda vibe just callin it out 🤷 .